vapi/tags/tag_association.go

```diff
@@ -93,7 +99,10 @@ func (c *Manager) AttachTagToMultipleObjects(ctx context.Context, tagID string,
 	}{ids}
 
 	url := c.Resource(internal.AssociationPath).WithID(id).WithAction("attach-tag-to-multiple-objects")
-	return c.Do(ctx, url.Request(http.MethodPost, spec), nil)
```

Comment:
just an error

Reply:
Resolved in https://github.com/vmware/govmomi/commit/d14cbf39ee092c1a12388ab5b8009bc3c443c898.